CSVSource/README.md

```diff
@@ -16,6 +16,8 @@ In this way, in cases where the extension is not running during file updates, it
 In addition to handling delimiter-separated fields, this connector can extract fields from fixed positions in the record.
 This is configured as described below.
 
+When running this connector as a docker image, the file path is dynamically converted to a UNIX file path. This means that the same configuration can be used for both docker and non-docker runtimes.
```

Comment:
This is simply not true.  If the configuration is a windows one, it'll be converted to unix.  The other way is not true.

Reply:
done